sqlstore/src/main/resources/db/migration/V8844__paritions_for_platform.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE Platform_Sizes (
+  platform_platformId bigint(20) NOT NULL,
+  partitionSize int NOT NULL,
+  PRIMARY KEY (platform_platformId, partitionSize),
+  CONSTRAINT fk_platform_size_platform FOREIGN KEY (platform_platformId) REFERENCES Platform (platformId)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO Platform_Sizes(platform_platformId, partitionSize)
+  SELECT DISTINCT platform, COUNT(*) AS c FROM SequencerPartitionContainer JOIN SequencerPartitionContainer_Partition ON containerId = container_containerId GROUP BY containerId;
```

Comment:
Some default sizes should be inserted for the platforms that are added in mainline migrations as well, in case of migrating to a new DB, the demo Docker container, etc

Reply:
Okay. I've updated the migration to include a bunch of of what it used to guess. No idea if those are right for many of the older platforms.